Source/Controllers/DataControllers/SPTableData.m

```diff
@@ -998,7 +998,7 @@ - (NSDictionary *) informationForView:(NSString *)viewName
 
 		// Add the column index and name
 		[tableColumn safeSetObject:[NSString stringWithFormat:@"%llu", (unsigned long long)[tableColumns count]] forKey:@"datacolumnindex"];
-		[tableColumn safeSetObject:[NSString stringWithString:[resultRow safeObjectForKey:@"Field"]] forKey:@"name"];
+		[tableColumn safeSetObject:[NSString stringWithString:safeString([resultRow safeObjectForKey:@"Field"])] forKey:@"name"];
```

Comment:
I don't understand the `stringWithString` part - why not `[tableColumn safeSetObject:[resultRow safeObjectForKey:@"Field"] forKey:@"name"];`?